src/include/libpmemset.h

```diff
@@ -70,24 +72,158 @@ extern "C" {
 #define PMEMSET_E_MAP_LENGTH_UNSET			(-200030)
 #define PMEMSET_E_SOURCE_FILE_IS_TOO_SMALL  (-200031)
 
-/* pmemset setup */
+/* source from file flags */
+
+#define PMEMSET_SOURCE_FILE_CREATE_ALWAYS		(1U << 0)
```

Comment:
Defines that are used only by single (or multiple similar functions which are next to each other) functions should be next to this function(s).